### ISSUE TITLE
【EESS】New submissions for Wed,  3 Jan 24

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#1048]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
### Identification of Additive Continuous-time Systems in Open and  Closed-loop
 - **Authors:** Rodrigo A. González, Koen Classens, Cristian R. Rojas, James S. Welsh, Tom Oomen
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2401.01263
 - **Pdf link:** https://arxiv.org/pdf/2401.01263
 - **Abstract**
 When identifying electrical, mechanical, or biological systems, parametric continuous-time identification methods can lead to interpretable and parsimonious models when the model structure aligns with the physical properties of the system. Traditional linear system identification may not consider the most parsimonious model when relying solely on unfactored transfer functions, which typically result from standard direct approaches. This paper presents a novel identification method that delivers additive models for both open and closed-loop setups. The estimators that are derived are shown to be generically consistent, and can admit the identification of marginally stable additive systems. Numerical simulations show the efficacy of the proposed approach, and its performance in identifying a modal representation of a flexible beam is verified using experimental data.
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Federated Multi-View Synthesizing for Metaverse
 - **Authors:** Yiyu Guo, Zhijin Qin, Xiaoming Tao, Geoffrey Ye Li
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.00859
 - **Pdf link:** https://arxiv.org/pdf/2401.00859
 - **Abstract**
 The metaverse is expected to provide immersive entertainment, education, and business applications. However, virtual reality (VR) transmission over wireless networks is data- and computation-intensive, making it critical to introduce novel solutions that meet stringent quality-of-service requirements. With recent advances in edge intelligence and deep learning, we have developed a novel multi-view synthesizing framework that can efficiently provide computation, storage, and communication resources for wireless content delivery in the metaverse. We propose a three-dimensional (3D)-aware generative model that uses collections of single-view images. These single-view images are transmitted to a group of users with overlapping fields of view, which avoids massive content transmission compared to transmitting tiles or whole 3D models. We then present a federated learning approach to guarantee an efficient learning process. The training performance can be improved by characterizing the vertical and horizontal data samples with a large latent feature space, while low-latency communication can be achieved with a reduced number of transmitted parameters during federated learning. We also propose a federated transfer learning framework to enable fast domain adaptation to different target domains. Simulation results have demonstrated the effectiveness of our proposed federated multi-view synthesizing framework for VR content delivery.
### SASA: Saliency-Aware Self-Adaptive Snapshot Compressive Imaging
 - **Authors:** Yaping Zhao, Edmund Y. Lam
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2401.00875
 - **Pdf link:** https://arxiv.org/pdf/2401.00875
 - **Abstract**
 The ability of snapshot compressive imaging (SCI) systems to efficiently capture high-dimensional (HD) data depends on the advent of novel optical designs to sample the HD data as two-dimensional (2D) compressed measurements. Nonetheless, the traditional SCI scheme is fundamentally limited, due to the complete disregard for high-level information in the sampling process. To tackle this issue, in this paper, we pave the first mile toward the advanced design of adaptive coding masks for SCI. Specifically, we propose an efficient and effective algorithm to generate coding masks with the assistance of saliency detection, in a low-cost and low-power fashion. Experiments demonstrate the effectiveness and efficiency of our approach. Code is available at: https://github.com/IndigoPurple/SASA
### Improving the Stability of Diffusion Models for Content Consistent  Super-Resolution
 - **Authors:** Lingchen Sun, Rongyuan Wu, Zhengqiang Zhang, Hongwei Yong, Lei Zhang
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2401.00877
 - **Pdf link:** https://arxiv.org/pdf/2401.00877
 - **Abstract**
 The generative priors of pre-trained latent diffusion models have demonstrated great potential to enhance the perceptual quality of image super-resolution (SR) results. Unfortunately, the existing diffusion prior-based SR methods encounter a common problem, i.e., they tend to generate rather different outputs for the same low-resolution image with different noise samples. Such stochasticity is desired for text-to-image generation tasks but problematic for SR tasks, where the image contents are expected to be well preserved. To improve the stability of diffusion prior-based SR, we propose to employ the diffusion models to refine image structures, while employing the generative adversarial training to enhance image fine details. Specifically, we propose a non-uniform timestep learning strategy to train a compact diffusion network, which has high efficiency and stability to reproduce the image main structures, and finetune the pre-trained decoder of variational auto-encoder (VAE) by adversarial training for detail enhancement. Extensive experiments show that our proposed method, namely content consistent super-resolution (CCSR), can significantly reduce the stochasticity of diffusion prior-based SR, improving the content consistency of SR outputs and speeding up the image generation process. Codes and models can be found at {https://github.com/csslc/CCSR}.
### Efficient Parallel Audio Generation using Group Masked Language Modeling
 - **Authors:** Myeonghun Jeong, Minchan Kim, Joun Yeop Lee, Nam Soo Kim
 - **Subjects:** Audio and Speech Processing (eess.AS); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.01099
 - **Pdf link:** https://arxiv.org/pdf/2401.01099
 - **Abstract**
 We present a fast and high-quality codec language model for parallel audio generation. While SoundStorm, a state-of-the-art parallel audio generation model, accelerates inference speed compared to autoregressive models, it still suffers from slow inference due to iterative sampling. To resolve this problem, we propose Group-Masked Language Modeling~(G-MLM) and Group Iterative Parallel Decoding~(G-IPD) for efficient parallel audio generation. Both the training and sampling schemes enable the model to synthesize high-quality audio with a small number of iterations by effectively modeling the group-wise conditional dependencies. In addition, our model employs a cross-attention-based architecture to capture the speaker style of the prompt voice and improves computational efficiency. Experimental results demonstrate that our proposed model outperforms the baselines in prompt-based audio generation.
### A Stochastic-MILP dispatch optimization model for Concentrated Solar  Thermal under uncertainty
 - **Authors:** Navid Mohammadzadeh, Huy Truong-Ba, Michael E. Cholette, Theodore A. Steinberg, Giampaolo Manzolini
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2401.01133
 - **Pdf link:** https://arxiv.org/pdf/2401.01133
 - **Abstract**
 Concentrated Solar Thermal (CST) offers a promising solution for large-scale solar energy utilization as Thermal Energy Storage (TES) enables electricity generation independently of daily solar fluctuations, shifting to high-priced electricity intervals. The development of dispatch planning tools is mandatory to account for uncertainties associated with solar irradiation and electricity price forecasts as well as limited storage capacity. This study proposes the Stochastic Mixed Integer Linear Program (SMILP) to maximize expected profit within a specified scenario space. The SMILP scenario space is generated by different Empirical Cumulative Distribution Function percentiles of the potential solar energy to accumulate in storage and the expected profit is estimated using the Sample Average Approximation (SAA) method. SMILP exhibits robust performance, however, its computational time poses a challenge. Thus, three heuristic solutions are developed which run a set of deterministic optimizations on different historical weather profiles to generate candidate dispatching plans (DPs). The candidate DP with the best average performance on all profiles is then selected. The new methods were applied to a case study for a 115 MW CST plant in South Australia. When the historical database has a limited set of historical weather profiles, the SMILP achieves 6% to 9% higher profit than the closest benchmark when the DP is applied to novel weather conditions. With a large historical weather data, the performance of SMILP and Heuristic-2 becomes nearly identical because the SMILP can only utilize a limited number of trajectories for optimization without becoming computationally infeasible. In this case, Heuristic-2 emerges a practical alternative, since it provides similar average profit in a reasonable time (saving about 7 hours in computing time).
### HAAQI-Net: A non-intrusive neural music quality assessment model for  hearing aids
 - **Authors:** Dyah A. M. G. Wisnu, Epri Pratiwi, Stefano Rini, Ryandhimas E. Zezario, Hsin-Min Wang, Yu Tsao
 - **Subjects:** Audio and Speech Processing (eess.AS); Machine Learning (cs.LG); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2401.01145
 - **Pdf link:** https://arxiv.org/pdf/2401.01145
 - **Abstract**
 This paper introduces HAAQI-Net, a non-intrusive deep learning model for music quality assessment tailored to hearing aid users. In contrast to traditional methods like the Hearing Aid Audio Quality Index (HAAQI), HAAQI-Net utilizes a Bidirectional Long Short-Term Memory (BLSTM) with attention. It takes an assessed music sample and a hearing loss pattern as input, generating a predicted HAAQI score. The model employs the pre-trained Bidirectional Encoder representation from Audio Transformers (BEATs) for acoustic feature extraction. Comparing predicted scores with ground truth, HAAQI-Net achieves a Longitudinal Concordance Correlation (LCC) of 0.9257, Spearman's Rank Correlation Coefficient (SRCC) of 0.9394, and Mean Squared Error (MSE) of 0.0080. Notably, this high performance comes with a substantial reduction in inference time: from 62.52 seconds (by HAAQI) to 2.71 seconds (by HAAQI-Net), serving as an efficient music quality assessment model for hearing aid users.
